sdk/purview/azure-purview-datamap/pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.azure-sdk-build]
+pylint = false
```

Comment:
Why is this check turned off?

Reply:
Suggested by @msyyc  to bypass a failure for pylink.